awx/settings/development.py

```diff
@@ -105,8 +92,28 @@
 AWX_DISABLE_TASK_MANAGERS = False
 # ======================!!!!!!! FOR DEVELOPMENT ONLY !!!!!!!=================================
 
-if 'sqlite3' not in DATABASES['default']['ENGINE']:  # noqa
-    DATABASES['default'].setdefault('OPTIONS', dict()).setdefault('application_name', f'{CLUSTER_HOST_ID}-{os.getpid()}-{" ".join(sys.argv)}'[:63])  # noqa
+# Store a snapshot of default settings at this point before loading any
+# customizable config files.
+#
+###############################################################################################
+#
+#  Any settings defined after this point will be marked as as a read_only database setting
+#
+################################################################################################
```

Comment:
I don't love the wording here, because everything "after" should be the includes of other files. What it really should _mean_ is that no ordinary settings should come after this point. After this, only logic for includes should be done. Then anything defined by the includes will result in read-only settings.